src/fe/inf_fe_static.C

```diff
@@ -304,18 +304,18 @@ void InfFE<Dim,T_radial,T_map>::compute_data(const FEType & fet,
 #ifdef LIBMESH_USE_COMPLEX_NUMBERS
 
   // assumption on time-harmonic behavior
-  const short int sign (-1);
+  Number sign (0,1.);
```

Comment:
Should we rename this sign_i, if it's now encompassing both parts of ```sign*i``` in the comment below?